src/main/java/Cassandra.java

```diff
@@ -0,0 +1,109 @@
+import java.util.ArrayList;
```

Comment:
It is good that import classes are listed explicitly, giving excellent documentation value for class at hand and making the class easier to comprehend and maintain.